static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js

```diff
@@ -25,12 +25,25 @@ define([
     var GustyleComcontent = function ($adSlot, params) {
         this.$adSlot = $adSlot;
         this.params  = params;
+        this.contentPosition = {
+            bottom: 'bottom: 30px;',
+            top: 'top: 30px;'
+        };
+        this.fontSize = {
```

Comment:
Yeah you are right. I just followed the pattern used by fluid ads but now I realised that it's not needed here.